packages/api/src/mikro-orm/migrations/Migration20240830112400.ts

```diff
@@ -0,0 +1,7 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20240830112400 extends Migration {
+    async up(): Promise<void> {
+        this.addSql('alter table "TargetGroup" add column "isTestList" boolean null;');
```

Comment:
This should not be nullable. You can set existing values to `false` since there are no existing test lists because it is a new feature.

Reply:
Drop the default value after setting all rows to false 🙏🏻